src/main/java/seedu/address/model/client/ClientPhone.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.model.client;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the phone of the client. This class is modelled after the Phone class in the Person package of AB3
+ */
+public class ClientPhone {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Phone numbers should only contain numbers, and it should be 8-10 digits";
+    public static final String VALIDATION_REGEX = "\\d{8,10}";
```

Comment:
There are phone numbers then do not fall within the range of 8-10 digits; leaving it as 3 or more digits would be fine

Reply:
Noted and changed